pragma/publisher/client.py

```diff
@@ -79,3 +87,174 @@ def fetch_sync(self) -> List[any]:
             data = fetcher.fetch_sync()
             results.extend(data)
         return results
+
+
+class PragmaAPIClient(PragmaClient):
+
+    api_base_url=os.getenv("PRAGMA_API_BASE_URL"),
+    api_key=os.getenv("PRAGMA_API_KEY"),
```

Comment:
add in constructor

Reply:
Since only the base_url differ when you choose dev or prod, and since we are already passing the base_url as a parameter in the constructor, I don't see why we need to specify the env here, unless it's needed somewhere else